code/src/MovieDetails.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import { Route, Link, useParams } from 'react-router-dom';
+import { apiKey } from './Key';
+import { BackButton } from './BackButton';
+import 'moviedetails.css';
+
+
+export const MovieDetails = () => {
+  const { movieId } = useParams();
+  const DETAILS_API = `https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}&language=en-US`;
+  const [movie, setMovie] = useState({});
+
+  useEffect(() => {
+    fetch(DETAILS_API)
+      .then((res) => res.json())
+      .then((json) => setMovie(json))
+  }, [DETAILS_API, movieId]);
+
+    return (
+      <div key={movie.id}>
```

Comment:
No need for `key` over here, we don't perform any loop

Reply:
Fixed!